scripts/poc906_deploy_cryptopunks_nft.py

```diff
@@ -0,0 +1,277 @@
+from brownie import(
+    accounts,
+    ERC721,
+    ERC20,
+    LendingPoolCore,
+    LendingPoolPeripheral,
+    CollateralVaultCore,
+    CryptoPunksVaultCore,
+    CollateralVaultPeripheral,
+    LoansCore,
+    Loans,
+    LiquidationsCore,
+    LiquidationsPeripheral,
+    LiquidityControls,
+    web3
+)
+
+
+def dev():
+    owner = accounts[0]
+
+    cool_cats_instance = ERC721.at('0x3194cBDC3dbcd3E11a07892e7bA5c3394048Cc87')
+    hashmasks_instance = ERC721.at('0x602C71e4DAC47a042Ee7f46E0aee17F94A3bA0B6')
+    kennel_instance = ERC721.at('0xE7eD6747FaC5360f88a2EFC03E00d25789F69291')
+    doodles_instance = ERC721.at('0x6951b5Bd815043E3F842c1b026b0Fa888Cc2DD85')
+    wow_instance = ERC721.at('0xe0aA552A10d7EC8760Fc6c246D391E698a82dDf9')
+    mutant_instance = ERC721.at('0X6b4BDe1086912A6Cb24ce3dB43b3466e6c72AFd3')
+    veefriends_instance = ERC721.at('0x9E4c14403d7d9A8A782044E86a93CAE09D7B2ac9')
+    pudgypenguins_instance = ERC721.at('0xcCB53c9429d32594F404d01fbe9E65ED1DCda8D9')
+    bayc_instance = ERC721.at('0x3981B4737bc9796A1Bf1b23bBB47Ec4104862f4f')
+    wpunks_instance = ERC721.at('0x168b6B69003Fa56ba2E243ea892f5cf2D09d049a')
+    cryptopunks_instance = ERC721.deploy({"from": owner})
+    print(f"cryptopunks address={cryptopunks_instance.address}")
+
+    collateral_vault_core = CollateralVaultCore.at('0x2c15A315610Bfa5248E4CbCbd693320e9D8E03Cc')
+    liquidations_core = LiquidationsCore.at('0x26f15335BB1C6a4C0B660eDd694a0555A9F1cce3')
+    loans_core_weth = LoansCore.at('0xe65A7a341978d59d40d30FC23F5014FACB4f575A')
+    lending_pool_peripheral_weth = LendingPoolPeripheral.at('0x7a3d735ee6873f17Dbdcab1d51B604928dc10d92')
+    liquidity_controls = LiquidityControls.at('0xed00238F9A0F7b4d93842033cdF56cCB32C781c2')
+    weth = ERC20.at('0x420b1099B9eF5baba6D92029594eF45E19A04A4A')
+
+    cryptopunks_vault_core = CryptoPunksVaultCore.deploy(cryptopunks_instance, {"from": owner})
+    print(f"cryptopunks_vault_core address={cryptopunks_vault_core.address}")
+
+    collateral_vault_peripheral = CollateralVaultPeripheral.deploy(
+        collateral_vault_core,
+        {"from": owner}
+    )
+    print(f"collateral_vault_peripheral address={collateral_vault_peripheral.address}")
+
+    loans_peripheral_weth = Loans.deploy(
+        1000000,
+        31 * 86400,
+        web3.toWei(10000, "ether"),
+        24 * 60 * 60,
+        loans_core_weth,
+        lending_pool_peripheral_weth,
+        collateral_vault_peripheral,
+        {"from": owner}
+    )
+    print(f"loans_peripheral_weth address={loans_peripheral_weth.address}")
+
+    liquidations_peripheral = LiquidationsPeripheral.deploy(
+        liquidations_core,
+        172800,
+        1296000,
+        1296000,
```

Comment:
Let's use the normal 2 days for these periods
```suggestion
        172800,
        172800,
        172800,
```